Networking/Networking/Model/WordPressApiError.swift

```diff
@@ -0,0 +1,61 @@
+import Foundation
+
+/// WordPress API Request Error
+///
+public enum WordPressApiError: Error, Decodable, Equatable {
```

Comment:
We already have an enum for WordPress.com Request Errors (`DotcomError`), for me to understand, do these two enums have different purposes?

Reply:
This is for the core API errors, which might come from WordPress.com (`wpcom/v2`, `wp/v2`) or directly from a self-hosted site (via `WordPressOrgRequest`). There might be some overlap, but error codes might not necessarily match.